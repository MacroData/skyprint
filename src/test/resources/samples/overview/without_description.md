# The Simplest API

# GET /message

