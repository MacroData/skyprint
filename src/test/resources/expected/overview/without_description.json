{
  "name": "The Simplest API",
  "description": ""
}
